packages/core/src/components/icon/_icon.scss

```diff
@@ -22,6 +22,10 @@
   > svg {
     // prevent extra vertical whitespace
     display: block;
+    // Allow width and height to be set on the span in css. This is ignored if the
+    // span does not have an explicit size set.
```

Comment:
how is it ignored? there's no logic that ignores it, we're relying on browser implementations it seems.